VERSION

```diff
@@ -0,0 +1 @@
+0.0.0-master
```

Comment:
It sounds good to move the version here, but I'm still confused about how we are going format the release version. We need to agree on it and I'd rather have this PR reflecting such a format.

Reply:
So the way other operators and the Operator SDK are versioning their projects is the following. The first release will be `0.0.1` because we still consider the project in an alpha status. Once released, the version is changed to `0.0.1+git` to indicate that there are changes on top of `0.0.1`. Then, `0.0.2` is released and the version immediately changed to `0.0.2+git`. This is the versioning strategy that similar projects are using and we plan to use.

Reply:
I don't understand what the digits mean if you do. Bumping the last version is supposed to indicate that it is a bug-fix release, but what you are saying does not seem to indicate that.

https://semver.org

Reply:
Makes sense... We'll then start with an initial version `0.1.0`. 

- `0`: indicates initial development. Anything may change at any time. The public API should not be considered stable.
- `1`: first set of features
- `0`: no bug fixes yet.

Reply:
https://github.com/pravega/pravega-operator/issues/53

Reply:
This has been merged now, but this doesn't really solve the initial problem because although the version number is OUTSIDE the GO code, it's still part of the actual code in the repository.  A much better way that I see a lot of projects using now is to calculate the SemVer from GIT tags.